### ISSUE TITLE
Error when apply sjoin() function

### ISSUE THREAD

**Issue by Yzyz-99** [geopandas/dask-geopandas#243]
Read geojson file using geopandas and convert to dask_geopandas
`bin_df = dask_geopandas.from_geopandas(bin_df, npartitions=28)`

Dask dataframe
`ddf = dd.read_csv(
        csv_file,
        usecols=['LATITUDE','LONGITUDE'], engine='python')`
`points = points.set_crs(4326)`

Convert dask dataframe to dask_geopandas GeoDataFrame
`points = dask_geopandas.from_dask_dataframe(ddf,
                          geometry=dask_geopandas.points_from_xy(ddf,"LONGITUDE","LATITUDE"),
                          )`

sjoin the datasets
`point_in_bin = bin_df.sjoin(points, how='inner', predicate='intersects')`

It gives me 
TypeError: sjoin() got an unexpected keyword argument 'predicate'

**Comment by martinfleis** [comment]
It seems that you are not using the latest version of geopandas and dask_geopandas. Either try updating or use `op='intersects' instead of `predicate`.

**Comment by Yzyz-99** [comment]
Both my geopandas and dask_geopandas are the latest version:
geopandas:
`geopandas            0.12.2`

dask_geopandas 
`dask-geopandas       0.3.0`

Even if I use `op='intersects'`, it still gives me the same error

**Comment by martinfleis** [comment]
Can you show the full traceback?

**Comment by Yzyz-99** [comment]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[c:\Users\User\OneDrive\Desktop\TM](file:///C:/Users/User/OneDrive/Desktop/TM) R&D intern\Untitled-1.ipynb Cell 14 in ()
----> [1](vscode-notebook-cell:/c%3A/Users/User/OneDrive/Desktop/TM%20R%26D%20intern/Untitled-1.ipynb#X20sZmlsZQ%3D%3D?line=0) point_in_bin = dask_geopandas.sjoin(bin_df, points, how='inner', predicate='intersects')

File [c:\Users\User\anaconda3\envs\internship\lib\site-packages\dask_geopandas\sjoin.py:60](file:///C:/Users/User/anaconda3/envs/internship/lib/site-packages/dask_geopandas/sjoin.py:60), in sjoin(left, right, how, predicate, **kwargs)
     57     right = from_geopandas(right, npartitions=1)
     59 name = "sjoin-" + tokenize(left, right, how, predicate)
---> 60 meta = geopandas.sjoin(left._meta, right._meta, how=how, predicate=predicate)
     62 if left.spatial_partitions is not None and right.spatial_partitions is not None:
     63     # Spatial partitions are known -> use them to trim down the list of
     64     # partitions that need to be joined
     65     parts = geopandas.sjoin(
     66         left.spatial_partitions.to_frame("geometry"),
     67         right.spatial_partitions.to_frame("geometry"),
     68         how="inner",
     69         predicate="intersects",
     70     )

TypeError: sjoin() got an unexpected keyword argument 'predicate'

**Comment by Yzyz-99** [comment]
Yes, it is. Even if it is an older version and I try to use `op`, it also gives the same error.

**Comment by martinfleis** [comment]
That is because we capture it here https://github.com/geopandas/dask-geopandas/blob/474c6c36e8c54806970b117ed24af98b276c2a09/dask_geopandas/sjoin.py#L43-L50

I honestly think that your installation of geopandas that is imported here is not 0.12.2. Can you try running the notebook outside of VSCode? It can sometimes behave funny around environments.

**Comment by Yzyz-99** [comment]
It solved when I try to reinstall the package. Thank You.
